packages/brand-moment/src/BrandMoment.scss

```diff
@@ -1,9 +1,15 @@
 @import "~@kaizen/design-tokens/sass/color-vars";
 @import "~@kaizen/design-tokens/sass/spacing-vars";
+@import "~@kaizen/design-tokens/sass/layout";
+
+.container {
+  max-width: 1392px;
```

Comment:
Sure you don't want to use: https://github.com/cultureamp/kaizen-design-system/blob/master/packages/design-tokens/tokens/layout.json#L4 ?

Reply:
oooh, thanks!